tests/unittests/Foundation/NSCacheTests.mm

```diff
@@ -99,35 +97,45 @@ - (void)cache:(NSCache*)cache willEvictObject:(id)object {
     NSCache* cache = nil;
     ASSERT_NO_THROW(cache = [[NSCache new] autorelease]);
     ASSERT_NO_THROW(cache.totalCostLimit = 499500); // 0 + 1 + ... + 998 + 999
-    for(int i = 0; i < 1000; ++i) {
+    for (int i = 0; i < 1000; ++i) {
         EXPECT_NO_THROW([cache setObject:@(i) forKey:@(i) cost:i]);
     }
 
     // we're still under cost limit, so all objects should be alive.
     EXPECT_OBJCNE(nil, [cache objectForKey:@(0)]);
     EXPECT_OBJCNE(nil, [cache objectForKey:@(600)]);
 
-    // This should evict the first n objects whose costs add up to ~1000. That's 1-45. 1-44 cost 990, which isn't enough; 45 costs 45, and puts us at 1035.
+    // This should evict the first n objects whose costs add up to ~1000. That's 1-45. 1-44 cost 990, which isn't enough; 45 costs 45, and
+    // puts us at 1035.
     EXPECT_NO_THROW([cache setObject:@(1000) forKey:@(1000) cost:1000]);
     // skip item 0: it just got refreshed and doesn't cost anything anyway.
-    for(int i = 1; i <= 45; ++i) {
+    for (int i = 1; i <= 45; ++i) {
         EXPECT_OBJCEQ_MSG(nil, [cache objectForKey:@(i)], "item %d", i);
     }
 
     EXPECT_OBJCNE(nil, [cache objectForKey:@(46)]);
     EXPECT_OBJCNE(nil, [cache objectForKey:@(1000)]);
 }
 
+// TODO: This segfaults on OSX after the end of the test
+// Thought it might've have had to do with leaked objects, but an extra release on
+// cache, lastEvictedCache, lastEvictedObject, and delegate, don't seem to fix it
+// Seems doubtful that it's an over-release, given that this passes on Windows
+#if TARGET_OS_WIN32
```

Comment:
I concur. With both of those; although DISABLED_TEST is easy enough but to be uniform it could be nice. Can you at least open an issue and put the number here?